### PR TITLE
remove forwardRef and design change

### PR DESCRIPTION
**Pull request by reshav-abraham**

### CODE REVIEW

**Comment by reshav-abraham** [comment]
Lol this is so much cleaner and palatable. 😂 Just goes to show code can be as complicated and as simple as you want it to be

File: client/components/MovieGrid.js [hunk 1]
```diff
@@ -1,61 +1,35 @@
...
-});
+export const MovieGrid = props => {
+    return (
+            <Grid style={{height:"700px", width:"1200px", overflow: "scroll"}}>
+                <div class="ui celled grid">
+                <Grid.Row>{props.fields.map(field => <Grid.Column width={4}><b>{field}</b></Grid.Column>)}</Grid.Row>
+                {
+                    // ["Movie", "Overview", "Release Date", "Language"]
+                    props.rowData.map(row => 
+                                            <Grid.Row>
```
Comment (fkotsian) [inline client/components/MovieGrid.js:18]: 😮

File: client/components/MovieGrid.js [hunk 2]
```diff
@@ -1,61 +1,35 @@
...
+                                                {/* <Segment> */}
+                                                    {props.fields.map(
+                                                        field =>
+                                                        <Grid.Column width={4}>
```
Comment (fkotsian) [inline client/components/MovieGrid.js:22]: 👏 😃

File: client/index.js [hunk 3]
```diff
@@ -1,29 +1,39 @@
...
-
-  function searchTopMovies() {
-    movieGridRef.current.searchTopMovies();
+    getTopMovies();
+  }, []);
+
+  function getTopMovies(){
+    let movieData = [];
+    getTopMoviesApi().then(data => {
+        data.results.forEach((x, i) => movieData.push({"Movie": x.title, "Overview": x.overview, "Release Date": x.release_date, "Language": x.original_language}));
```
Comment (fkotsian) [inline client/index.js:24]: love it! clean :D

File: client/index.js [hunk 4]
```diff
@@ -36,20 +46,14 @@ function App() {
...
-          {/* <button 
-            class="ui button"
-            onClick={searchMovies}
-            style={{"backgroundColor":"purple", color:"white", "marginLeft": "10px", "height":"55px", width:"55px", textAlign:"center"}}
-            >
-              <Icon size="big" name="angle right icon" style={{marginLeft:"-8.5px"}}/>
-            </button> */}
           <button class="ui primary button" style={{"backgroundColor":"cyan", "color":"Grey", "marginLeft": "50px", "height":"55px"}} 
-            onClick={searchTopMovies}>
+            onClick={getTopMovies}>
```
Comment (fkotsian) [inline client/index.js:50]: nice! 🌲 ah.... so fresh and so clean clean :D

**Review by fkotsian — APPROVED** [review]
👍 !

Wow - yeah really nice! Just goes to show that a little change can go a long way - love architecture/design!

**Pull request merged**